ibm_catalog.json

```diff
@@ -22,30 +22,39 @@
-          "title": "Creates an IBM Cloud Event Notifications instance",
+          "title": "Event Notifications instance",
           "description": "For more details on an IBM Cloud Event Notifications instance, [see here](https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-about)."
         },
         {
-          "title": "Creates Key Rings and Keys to encrypt data",
-          "description": "For more details on managing encryption using an existing Key Management Services (Key Protect or HPCS) instance, [see here](https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-managing-encryption)."
+          "title": "KMS Encryption",
+          "description": "Optionally you can enable KMS (Key Management Services) [encryption](https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-managing-encryption) of the Event Notifications instance and the Object Storage bucket using either a newly created key or an existing one."
         },
         {
-          "title": "Collect events that fail delivery",
-          "description": "Connect an existing IBM Cloud Object Storage (COS) service instance and creates a new bucket to collect events that fail delivery. For more details on collecting failed events [see here](https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-cfe-integrations)."
+          "title": "Object Storage",
+          "description": "Creates and configures an Object Storage bucket to collect events that fail delivery. You can provide an existing Cloud Object Storage (COS) instance or use the [Cloud automation for Object Storage](https://cloud.ibm.com/catalog/7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3/architecture/deploy-arch-ibm-cos-68921490-2778-4930-ac6d-bae7be6cd958-global) for creating a new instance. For more details on collecting failed events, [learn more](https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-cfe-integrations)."
+        },
+        {
+          "title": "Observability",
+          "description": "This solution can leverage [Cloud automation for Observability](https://cloud.ibm.com/catalog/7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3/architecture/deploy-arch-ibm-observability-a3137d28-79e0-479d-8a24-758ebd5a0eab-global) that supports configuring resources for logging, monitoring and activity tracker event routing (optional)."
+        },
+        {
+          "title": "Service credentials",
+          "description": "Has the ability to create and configure [event notification service credentials](https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-service-credentials) for your instance. Optionally choose to configure [Cloud automation for Secrets Manager](https://cloud.ibm.com/catalog/7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3/architecture/deploy-arch-ibm-secrets-manager-6d6ebc76-7bbd-42f5-8bc7-78f4fabd5944-global) to create a IBM Cloud Secrets Manager instance to store the service credentials."
```

Comment:
Secrets Manager is not defined as a dependant DA here - its cant because it would introduce a circular dependency